File: 2984486(small)/TakaakiUmedu/5752104073297920/0/extracted/c2.rb

def calc(a)
	sum = 0
	a.each_with_index{|v, i|
		sum += (v - i) * (v - i)
	}
	Math.sqrt(sum / a.length)
end

T = gets.to_i
(1..T).each{|t|
	n = gets.to_i
	a = gets.strip.split(/ /).collect{|num| num.to_i}
	ave = calc(a)
	if ave < 350
		puts "Case \##{t}: BAD"
	else
		puts "Case \##{t}: GOOD"
	end
}

File: 2984486(small)/starsep/5752104073297920/0/extracted/c2.cpp
#include <cstdio>
bool solve()
{
	int n,q,p;
	int N;
	scanf("%d",&n);
	N=n;
	scanf("%d",&q);
	n--;
	while(n--)
		scanf("%d",&p);
	q++;
	return q>=N/2;
}
int main()
{
	int t;
	scanf("%d",&t);
	for(int i=1;i<=t;i++)
		printf("Case #%d: %s\n",i,solve() ? "BAD" : "GOOD");
}